quick/commands/base.py

```diff
@@ -75,17 +74,18 @@ def execute(self):
 
     def create_sub_parser(self, parent: SubParserAction):
         self.description = self.description or self.help
-        self.parser = parent.add_parser(
+        parser = parent.add_parser(
             self.name,
             description=self.description,
             help=self.help,
             formatter_class=QuickHelpFormatter,
         )
-        required = self.parser.add_argument_group("Required")
-        optional = self.parser.add_argument_group("Optional")
-        self.add_args(self.parser, required, optional)
+        required = parser.add_argument_group("Required")
+        optional = parser.add_argument_group("Optional")
+        self.add_args(parser, required, optional)
         self.add_common_args(optional)
-        self.parser.set_defaults(func=self)
+        parser.set_defaults(func=self)
+        self.parser = parser
```

Comment:
Just curious about the change here. Didn't it work before?

Reply:
It did work but the newest version of mypy wasn't happy with it :smile: